packages/dart/noports_core/lib/src/srv/srv_impl.dart

```diff
@@ -142,7 +142,9 @@ class SrvImplExec implements Srv<Process> {
       }
     });
 
-    await rvPortBound.future.timeout(Duration(seconds: 2));
+    await rvPortBound.future.timeout(Duration(seconds: 3));
+
+    await Future.delayed(Duration(milliseconds: 100));
```

Comment:
A short grace period which should mean that the child process doesn't encounter an exception from stderr.writeln